charger/switchsocket.go

```diff
@@ -121,6 +121,10 @@ func (c *switchSocket) Status() (api.ChargeStatus, error) {
 	}
 
 	// standby power mode
+	if c.currentPower == nil {
```

Comment:
if there's no currentPower, negative standypower is required. This change is not needed.